packages/ag-charts-community/src/chart/series/cartesian/scatterSeries.ts

```diff
@@ -225,6 +225,7 @@ export class ScatterSeries extends CartesianSeries<SeriesNodeDataContext<Scatter
         }
 
         nodeData.length = actualLength;
+        this.fireDataProcessed(dataModel, processedData);
```

Comment:
Should/could we move this to the super-class so all `CartesianSeries` have this behaviour after running `processData()`?

Reply:
I initially tried doing this ([ba1739ec7](https://github.com/ag-grid/ag-charts/commit/ba1739ec7d47391ac01639538dad7fb4704cc2f0)).

I cannot remember exactly what went wrong with this approach, but AFAIK the `Scale.convert()` function was not returning the expected result when called by the event handler in `errorBar.ts`. This could perhaps be because the axes are not fully initialised at this point. On the other hand, firing the event during the `createNodeData()` function gives the expected `convert()` return values.

That said, I think he should try to make this work by firing the event in `processData()` because it would be more robust.

Should I tried to resolve this within the scope of this PR or leave it as an action item for a follow-up?

Reply:
Actually, I've found another way to fire the event in the `CartesianSeries` base class ([a0c519a1](https://github.com/ag-grid/ag-charts/pull/258/commits/a0c519a1a556aa803aebd09a1cf338b7f73f00c0)). Resolving.